com_039_rw0_agricultural_trade_statistics/com_039_rw0_agricultural_trade_statistics_processing.py

```diff
@@ -78,3 +83,44 @@
 # save processed dataset to CSV
 processed_data_file = os.path.join(data_dir, dataset_name+'_edit.csv')
 df.to_csv(processed_data_file, index=False)
+
+'''
+Upload processed data to Carto
+'''
+logger.info('Uploading processed data to Carto.')
+
+# authenticate carto account
+CARTO_USER = os.getenv('CARTO_WRI_RW_USER')
+CARTO_KEY = os.getenv('CARTO_WRI_RW_KEY')
+set_default_credentials(username=CARTO_USER, base_url="https://{user}.carto.com/".format(user=CARTO_USER),api_key=CARTO_KEY)
+
+# upload data frame to Carto
+to_carto(df, dataset_name + '_edit', if_exists='replace', privacy="link")
+
+# set privacy to 'link' so table is accessible but not published
+update_privacy_table(dataset_name + '_edit', 'link')
```

Comment:
Not sure if `to_carto` has the parameter `privacy`, I didn't see it on their [reference](https://carto.com/developers/cartoframes/reference/). If so, we don't need to run `update_privacy_table`. If not, we should remove that from the line above.

Reply:
Right, I was experimenting with that option but it didn't work. Removed in 91dc2c295629e2559ca7593ac6e7df8aa35b58d1